web-marketplace/src/legacy-pages/Dashboard/MyCreditBatches/MyCreditBatches.tsx

```diff
@@ -35,6 +37,32 @@ export const MyCreditBatches = (): JSX.Element => {
     ? isOrganizationOwner || isOrganizationAdmin
     : isIssuer;
 
+  const { roleId, authorizationId } = getRoleAuthorizationIds({
+    type: 'organization',
+    currentUserRole: organizationRole,
+    authorizationName: 'can_manage_credit_issuance',
+  });
+
+  const hasOrgExecutionContext = Boolean(
+    isOrganizationDashboard &&
+      organizationDaoAddress &&
+      organizationRbamAddress &&
+      authorizationId &&
+      roleId,
+  );
+
+  const createBatchState = hasOrgExecutionContext
+    ? {
+        issuerAddress: selectedAccountAddress,
+        organizationDaoAddress,
+        organizationRbamAddress,
+        authorizationId,
+        roleId,
+      }
+    : selectedAccountAddress
+    ? { issuerAddress: selectedAccountAddress }
+    : undefined;
```

Comment:
we could just leave the state empty outside of org execution context
since the create batch code will just pick up the current user address instead